### ISSUE TITLE
Installation not working

### ISSUE THREAD

**Issue by flotscho** [Villarrealized/sonos-web#161]
Hey Guys,

I want to install the sonos- web but after like 2min i get this error:

Flo@Grundtner:/$ Flo@Grundtner:/$ sudo sonos-web install
Installing Sonos Web
✔ downloading installation files
✔ installing front-end dependencies
✖ build front-end application
error installation failed...check /root/.sonos-web/install.log for more details

Any ideas? Thank you!

**Comment by tiigaaa** [comment]
same here...
did you find a solution?

**Comment by breaderz88** [comment]
@flotscho @tiigaaa Hi guys I had same issues I spent a bit of time on this yesterday and I got it working for me. here is a link to my update https://drive.google.com/file/d/1xBba4aLde2wn6oxp_GXKpfjVvEAm2heW/view?usp=sharing. go to the server folder once downloaded on PC, launch cmd, and do npm start. It should work now. let me know.

**Comment by tiigaaa** [comment]
THANKS, i will try it 👍

**Comment by flotscho** [comment]
@breaderz88 thanks for your work. But can you please tell me where to put this downloaded folder (raspberry Linux) and how to start the installation out of it? Thanks!

**Comment by breaderz88** [comment]
@flotscho I used PC for this but it should still work(Hopefully) if you just add the folder to your desktop for the time being. then can you use bash to navigate the the server folder inside sonos then run the command npm start?

**Comment by flotscho** [comment]
Thanks, i did it already but now it stopped here, so whats next!?

![image](https://github.com/sonos-web/sonos-web/assets/128217671/8fd51f27-a6ff-4887-a818-d706bf1c4fd4)

**Comment by breaderz88** [comment]
after this is there anything when you go to browser 192.168.1.56? I normally launch the server and then go to http://192.168.1.70:5051 and it works fine. This maybe be an issue with Linux however as I used PC.

**Comment by flotscho** [comment]
Yeah there is the web interface but thats not really working right now. And when i close the shell window the connection get lost.

![image](https://github.com/sonos-web/sonos-web/assets/128217671/3c714907-9a96-4be2-874f-651cd479e94e)

**Comment by tiigaaa** [comment]
Hi, Im also on LINUX UBUNTU (from the shell) and get the same Error.
I can see all Sonos deviced, I can see my Favorites but I cant play them.

I wil try it on Wondows and report back :) Thanks so far.

**Comment by tuk90** [comment]
Just use the docker version that works properly, run it on the host network via: sudo docker run -d --network host --restart unless-stopped sonosweb/sonos-web
